hermes/bin/src/runtime_extensions/mod.rs

```diff
@@ -1,11 +1,15 @@
 //! Hermes runtime extensions
 
+use tracing::{span, Level};
+
 pub(crate) mod bindings;
 pub mod hermes;
 pub(crate) mod wasi;
 
 /// Advise Runtime Extensions of a new context
 pub(crate) fn new_context(ctx: &crate::runtime_context::HermesRuntimeContext) {
-    hermes::new_context(ctx);
-    wasi::new_context(ctx);
+    span!(Level::INFO, "Context Span", ctx = ?ctx).in_scope(|| {
```

Comment:
Probably I am missing something, but don't we need to enter the span `enter()` method to log everything we have under this span context ?

Reply:
The `in_scope` should work in a similar behavior as `enter()` 
https://docs.rs/tracing/latest/tracing/span/struct.EnteredSpan.html